### DISCUSSION TITLE
Finding root .tex-file with space : \begin {document}

### DISCUSSION THREAD

**Discussion by WaSaDu** [James-Yu/LaTeX-Workshop#3165] [Q&A] [answered]
I have noticed that the root file is not detected if there is a space before the {: <\begin<space>{document}>. After removing this space, it works. 

I think this is a bug, as LaTeX is not space sensitive at this point.

**Comment by James-Yu** [comment, marked as answer]
Just remove the space. Space before curly brace is not standard.
